### PR TITLE
Added explicit paths

### PR DESCRIPTION
**Pull request by jnolis**
Added explicit paths for the examples, since the default might be changing soon.

### CODE REVIEW

File: examples/rapids-comparison/.saturn/README.md [hunk 1]
```diff
@@ -1,6 +0,0 @@
-# Zero to RAPIDS in Minutes with NVIDIA GPUs + Saturn Cloud
```
Comment (jsignell) [inline examples/rapids-comparison/.saturn/README.md:1]: This one snuck in looks like

**Review by jsignell — APPROVED** [review]
LGTM

**Comment by jnolis** [comment]
@jsignell do you know why this might be stuck on linting/validating?

**Comment by jsignell** [comment]
Probably jut github being dumb yesterday. I'll kick it.

**Pull request merged**